packages/extension/src/components/DaSettings.vue

```diff
@@ -75,6 +77,10 @@ export default {
       this.$store.commit('ui/setShowOnlyNotReadPosts', val);
       this.refreshFeed();
     },
+    async toggleOpenInTab(val) {
```

Comment:
This function doesn't have to be `async`. Nothing `async` really going on inside it